module/module.go

```diff
@@ -559,3 +739,18 @@ func writeFile(filePath string, bytes []byte) error {
 func closeFile(file *os.File) {
 	_ = file.Close()
 }
+
+var camelingRegex = regexp.MustCompile("[0-9A-Za-z]+")
+
+func camelCase(src string) string {
```

Comment:
it doesnt cause cycles if the function stays in module system.

Aka `codegen` can import `CamelCase` from `module` BUT `module` CANNOT import `CamelCase` from `codegen`.